src/main/java/DynInt.java

```diff
@@ -0,0 +1,9 @@
+public class DynInt {
+    public static void main(String[] args) {
+        double radius = 4, height = 5;
+
+        double volume = 3.1416 * radius * radius * height;
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [CNT: Rough value of known constant found (CNT_ROUGH_CONSTANT_VALUE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#cnt-rough-value-of-known-constant-found-cnt-rough-constant-value).
It's recommended to use the predefined library constant for code clarity and better precision.

Reply:
thnx, it's kind of test pull request